website/content/commands/acl/auth-method/create.mdx

```diff
@@ -71,6 +65,8 @@ Usage: `consul acl auth-method create [options] [args]`
 
 #### Enterprise Options
 
+@include 'http_api_partition_options.mdx'
```

Comment:
Since partitions sit above namespaces in the Consul organizational hierarchy, I think it makes sense to show partition above namespace in the list. Thoughts?